test/cli/test_catalog.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from unittest.mock import patch, MagicMock
+from click.testing import CliRunner
+from ersilia.cli.commands.catalog import catalog_cmd
+
+@pytest.fixture
+def runner():
+    return CliRunner()
+
+@patch("ersilia.hub.content.catalog.ModelCatalog.airtable", return_value=None)
+@patch("ersilia.hub.content.catalog.ModelCatalog.local")
+@patch("ersilia.hub.content.catalog.ModelCatalog.hub")
+@patch("ersilia.hub.content.card.ModelCard.get", return_value={"model": "metadata"})
+@pytest.mark.parametrize(
+    "options, model",
+    [
+        (["--local"], None),
+        (["--hub"], None),  
+        (["eos3b5e", "--card"]), 
+        (["--card"], None),
+    ]
+)
+def test_catalog_command(
+    mock_model_card_get,
+    mock_model_catalog_hub,
+    mock_model_catalog_local,
+    mock_model_catalog_airtable,
+    runner,
+    options,
+    model,
+):
+
+    mock_model_card_instance = MagicMock()
+
+    if model:
+        mock_model_card_instance.invoke.return_value.exit_code = 0
+        mock_model_card_instance.invoke.return_value.output = (
+            f"Fetching model {model}: \n👍 Model {model} executed successfully!\n"
+        )
+
+    result = runner.invoke(catalog_cmd(), options)
+    assert result.exit_code == 0, f"Unexpected exit code: {result.exit_code}. Output: {result.output}"
+
+    if "--card" in options and len(options) == 1: 
+        mock_model_card_get.assert_not_called()
+    elif "--card" in options and model: 
+        mock_model_card_get.assert_called_once()
+    elif "--hub" in options:
+        mock_model_catalog_hub.assert_called_once()
+    elif "--browser" in options:
+        mock_model_catalog_airtable.assert_called_once()
+    elif "--hub" in options:
+        mock_model_catalog_hub.assert_called_once()
+    elif "--local" in options:
+        mock_model_catalog_local.assert_called_once()
```

Comment:
We can also add a test for an exception being raised if both `--hub`, and `--local` are present in options bec both shouldn't be specified together.